openrtb_ext/request.go

```diff
@@ -23,6 +23,7 @@ type ExtRequestPrebid struct {
 	SChains              []*ExtRequestPrebidSChain `json:"schains,omitempty"`
 	StoredRequest        *ExtStoredRequest         `json:"storedrequest,omitempty"`
 	Targeting            *ExtRequestTargeting      `json:"targeting,omitempty"`
+	Events               json.RawMessage           `json:"events,omitempty"`
```

Comment:
Can this be a strongly typed struct instead?

Reply:
It can, but it would be an empty struct, as the only possible value so far is `events: {}`.

Reply:
Ah. Is this like caching where the presence is the trigger?

Reply:
`ext.prebid.cache` needs a value though (`vastxml` and/or `bids`). 
I expect that in time, so will `ext.prebid.events`, but for now it works as you said.